src/libraries/System.Net.Quic/src/System/Net/Quic/Interop/msquic.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Quic
 {
     internal unsafe partial struct QUIC_BUFFER
     {
-        public Span<byte> Span => new(Buffer, (int)Length);
+        public readonly Span<byte> Span => new(Buffer, (int)Length);
```

Comment:
I don't remember exactly, but if this is generated code from MsQuic, it should be fixed there as well.

Reply:
right, I thought this was added manually by us (since the QUIC_BUFFER definition is in another auto-generated file). I will follow up to make the change in their repo as well.